### PR TITLE
Update docs to reflect new naming for UniFi integration

### PR DESCRIPTION
**Pull request by AngellusMortis**
## Proposed change

Accompanying [PR for renames in core repo](https://github.com/home-assistant/core/pull/59568). Idea is to clarify scope and use of the main UniFi integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/59568
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/unifi.markdown [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Ubiquiti UniFi
+title: UniFi Network
 description: Instructions on how to configure UniFi integration with UniFi Controller by Ubiquiti.
```
Comment (Kane610) [inline source/_integrations/unifi.markdown]:
```suggestion
description: Instructions on how to configure UniFi integration with UniFi Network Controller by Ubiquiti.
```

File: source/_integrations/unifi.markdown [hunk 2]
```diff
@@ -32,17 +32,21 @@ There is currently support for the following device types within Home Assistant:
 
 The user needs administrator privileges in order to control switches.
 
+### Understanding UniFi Naming
```
Comment (Kane610) [inline source/_integrations/unifi.markdown]: Put this in an FAQ at the bottom

File: source/_integrations/unifi.markdown [hunk 3]
```diff
@@ -32,17 +32,21 @@ There is currently support for the following device types within Home Assistant:
...
+
+Network management has always been Ubiquiti's main product and so UniFi for a while was always synonymous with their Network Controller application. However, UniFi has started branching and releasing other apps, like Protect, Talk and Access. [This post on the UniFi community](https://community.ui.com/questions/Clarifying-UniFi-Hardware-and-Software-Terminology/2557963a-e79d-4157-a78c-36d3f7b383fb) explains it pretty well. **This main UniFi Integration is _only_ for UniFi Network application. It will not interface with devices for other applications like Protect, Talk, Access.**
```
Comment (Kane610) [inline source/_integrations/unifi.markdown]:
```suggestion
Network management has always been Ubiquiti's main product and so UniFi for a while was always synonymous with their Network Controller application. However, UniFi has started branching and releasing other apps, like Protect, Talk and Access. [This post on the UniFi community](https://community.ui.com/questions/Clarifying-UniFi-Hardware-and-Software-Terminology/2557963a-e79d-4157-a78c-36d3f7b383fb) explains it pretty well. **This main UniFi Integration is _only_ for UniFi Network application.**
```
Lets remove keywords that can be used when searching. to wrongfully find this integration

**Review by Kane610 — CHANGES_REQUESTED** [review]

File: source/_integrations/unifi.markdown [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 title: UniFi Network
-description: Instructions on how to configure UniFi integration with UniFi Controller by Ubiquiti.
+description: Instructions on how to configure UniFi integration with UniFi Network Application by Ubiquiti.
```
Comment (Kane610) [inline source/_integrations/unifi.markdown]:
```suggestion
description: Instructions on how to configure UniFi integration with UniFi Network application by Ubiquiti.
```

File: source/_integrations/unifi.markdown [hunk 5]
```diff
@@ -119,3 +115,9 @@ logger:
     homeassistant.components.device_tracker.unifi: debug
     homeassistant.components.switch.unifi: debug
 ```
+
+## FAQ
+
+### Understanding UniFi Naming (UniFi Controller vs. UniFi Network Application)
+
+Network management has always been Ubiquiti's main product and so UniFi for a while was always synonymous with their "UniFi Controller" application. However, UniFi has started branching and releasing other apps, like Protect, Talk and Access. As a result, Ubiquiti has started rebranding "UniFi Controller" as the "UniFi Network Application". [This post on the UniFi community](https://community.ui.com/questions/Clarifying-UniFi-Hardware-and-Software-Terminology/2557963a-e79d-4157-a78c-36d3f7b383fb) explains it pretty well. **This main UniFi Integration is _only_ for the UniFi Network application.**
```
Comment (Kane610) [inline source/_integrations/unifi.markdown]:
```suggestion
Network management has always been Ubiquiti's main product and so UniFi for a while was always synonymous with their "UniFi Controller" application. However, UniFi has started branching and releasing other apps, like Protect, Talk and Access. As a result, Ubiquiti has started rebranding "UniFi Controller" as the "UniFi Network" application. [This post on the UniFi community](https://community.ui.com/questions/Clarifying-UniFi-Hardware-and-Software-Terminology/2557963a-e79d-4157-a78c-36d3f7b383fb) explains it pretty well. **This main UniFi Integration is _only_ for the UniFi Network application.**
```

**Review by Kane610 — CHANGES_REQUESTED** [review]
The Ubiquiti document references `UniFi [Name] application will now replace controller (e.g., UniFi Protect application)`

File: source/_integrations/unifi.markdown [hunk 6]
```diff
@@ -74,7 +74,7 @@ Try to get a wireless client to reconnect to the network.
 
 ### Service unifi.remove_clients
 
-Clean up clients on the UniFi Controller that has only been associated with the controller for a short period of time. The difference between first seen and last seen needs to be less than 15 minutes and the client can not have a fixed IP, hostname or name associated with it.
+Clean up clients on the UniFi Network Controller that has only been associated with the controller for a short period of time. The difference between first seen and last seen needs to be less than 15 minutes and the client can not have a fixed IP, hostname or name associated with it.
```
Comment (Kane610) [inline source/_integrations/unifi.markdown]:
```suggestion
Clean up clients on the UniFi Network application that has only been associated with the controller for a short period of time. The difference between first seen and last seen needs to be less than 15 minutes and the client can not have a fixed IP, hostname or name associated with it.
```
```suggestion
Clean up clients on the UniFi Network Controller that has only been associated with the controller for a short period of time. The difference between first seen and last seen needs to be less than 15 minutes and the client can not have a fixed IP, hostname or name associated with it.
```

File: source/_integrations/unifi.markdown [hunk 7]
```diff
@@ -34,15 +34,15 @@ The user needs administrator privileges in order to control switches.
...
+All configuration options are offered from the front end. Enter what UniFi Network integration you want to change options on and press the cog wheel. Some advanced options are available when "Advanced Mode" is enabled on your user profile page.
 
 ### Configuring Users
 
-The UniFi controller allows you to create multiple users on it besides the main administrator. If all you want to use is the device tracker then it is recommended that you create a limited user that has `read-only` permissions for the UniFi device tracker. If you want blocking of network access or POE control as well you would need to have 'admin' permissions.
+The UniFi Network controller allows you to create multiple users on it besides the main administrator. If all you want to use is the device tracker then it is recommended that you create a limited user that has `read-only` permissions for the UniFi Network device tracker. If you want blocking of network access or POE control as well you would need to have 'admin' permissions.
```
Comment (Kane610) [inline source/_integrations/unifi.markdown]:
```suggestion
The UniFi Network application allows you to create multiple users on it besides the main administrator. If all you want to use is the device tracker then it is recommended that you create a limited user that has `read-only` permissions for the UniFi Network device tracker. If you want blocking of network access or POE control as well you would need to have 'admin' permissions.
```

File: source/_integrations/unifi.markdown [hunk 8]
```diff
@@ -52,13 +52,13 @@ It is recommended that you run the UniFi controller in a dedicated virtual machi
 
 ## Presence detection
 
-This platform allows you to detect presence by looking at devices connected to a [Ubiquiti](https://ui.com/) [UniFi](https://unifi-network.ui.com/) controller. By default devices are marked as away 300 seconds after they were last seen.
+This platform allows you to detect presence by looking at devices connected to a [Ubiquiti](https://ui.com/) [UniFi Network](https://unifi-network.ui.com/) controller. By default devices are marked as away 300 seconds after they were last seen.
```
Comment (Kane610) [inline source/_integrations/unifi.markdown]:
```suggestion
This platform allows you to detect presence by looking at devices connected to a [Ubiquiti](https://ui.com/) [UniFi Network](https://unifi-network.ui.com/) application. By default devices are marked as away 300 seconds after they were last seen.
```

File: source/_integrations/unifi.markdown [hunk 9]
```diff
@@ -52,13 +52,13 @@ It is recommended that you run the UniFi controller in a dedicated virtual machi
...
 
 ### Troubleshooting and Time Synchronization
 
 Presence detection depends on accurate time configuration between Home Assistant and the UniFi controller.
 
-If Home Assistant and the UniFi controller are running on separate machines or VMs ensure that all clocks are synchronized. Failing to have synchronized clocks will lead to Home Assistant failing to mark a device as home.
+If Home Assistant and the UniFi Network controller are running on separate machines or VMs ensure that all clocks are synchronized. Failing to have synchronized clocks will lead to Home Assistant failing to mark a device as home.
```
Comment (Kane610) [inline source/_integrations/unifi.markdown]:
```suggestion
If Home Assistant and the UniFi Network application are running on separate machines or VMs ensure that all clocks are synchronized. Failing to have synchronized clocks will lead to Home Assistant failing to mark a device as home.
```

File: source/_integrations/unifi.markdown (hunk 6 above)
Comment (Kane610) [inline source/_integrations/unifi.markdown]:
```suggestion
Clean up clients on the UniFi Network application that has only been associated with the Network application for a short period of time. The difference between first seen and last seen needs to be less than 15 minutes and the client can not have a fixed IP, hostname or name associated with it.
```

File: source/_integrations/unifi.markdown [hunk 10]
```diff
@@ -104,7 +104,7 @@ Get entities reporting uptime per network client.
 
 ## Debugging integration
 
-If you have problems with UniFi or the integration you can add debug prints to the log.
+If you have problems with UniFi Network or the integration you can add debug prints to the log.
```
Comment (Kane610) [inline source/_integrations/unifi.markdown]:
```suggestion
If you have problems with the UniFi Network application or integration you can add debug prints to the log.
```

**Review by Kane610 — CHANGES_REQUESTED** [review]
Let's be consistent. Please double check if I missed anything

File: source/_integrations/unifi.markdown [hunk 11]
```diff
@@ -115,3 +115,9 @@ logger:
     homeassistant.components.device_tracker.unifi: debug
     homeassistant.components.switch.unifi: debug
 ```
+
+## FAQ
+
+### Understanding UniFi Naming (UniFi Controller vs. UniFi Network Application)
```
Comment (Kane610) [inline source/_integrations/unifi.markdown]:
```suggestion
### Understanding UniFi Naming (UniFi Network application is the UniFi Controller)
```

*Nearly there* There are at least two instances of controller I couldn't add a comment on in MQTT and troubleshooting

Comment (AngellusMortis) [reply source/_integrations/unifi.markdown]: Yep, I only committed your suggestions. Going through it in depth now in my editor.

**Review by Kane610 — APPROVED** [review]
Awesome

**Comment by Kane610** [comment]
Thanks @AngellusMortis 🎉

**Pull request merged**